### ISSUE TITLE
/blocklocker <line number> <text>is not working

### ISSUE THREAD

**Issue by YuanYuanOwO** [rutgerkok/BlockLocker#164]
if i use the command,it's always show `/bl reload`
and the sign's text won't be change

with
Paper-46 MC 1.20.1
```
> plugins
[21:42:55 INFO]: Server Plugins (1):
[21:42:55 INFO]: Bukkit Plugins:
[21:42:55 INFO]:  - BlockLocker
```

**Comment by YuanYuanOwO** [comment]
thanks for reply!
now it's worked.
But the command can still be found in the tab completion
![{4D2467C7-483D-40f0-9047-55B428EA7922}](https://github.com/rutgerkok/BlockLocker/assets/81153017/0434ed9e-7690-4d99-af79-fd2b79a3faab)

**Comment by rutgerkok** [comment]
Ah thanks, I'll have a look!

**Comment by PaguroCavaliere** [comment]
Not exactly, if I right click on the sign it doesn't change. Every time I have to replace the sign with shift + right click and manually write [private] + names. I think it's better with the command, can you put the command back or do so that by right-clicking I can directly modify my private?

**Comment by rutgerkok** [comment]
> do so that by right-clicking I can directly modify my private?

That's how it currently works. Could it be that some other plugin is preventing sign editing?
